webapp-django/crashstats/settings/base.py

```diff
@@ -6,6 +6,9 @@
 # repo. If you need to override a setting locally, use .env or environment
 # variables.
 
+# Ignore unused imports when linting. Otherwise flake8 balks at NPM_FILE_PATTERNS.
+# flake8: noqa: F401
```

Comment:
I couldn't get black and flake8 to work with adding a `noqa` for `NPM_FILE_PATTERNS`. Seems like my only option was to nix that error for the whole file. :disappointed: